src/Avalonia.Controls/Presenters/ItemVirtualizer.cs

```diff
@@ -99,9 +100,14 @@ public Size Extent
         {
             get
             {
-                return Vertical ?
-                    new Size(Owner.Panel.DesiredSize.Width, ExtentValue) :
-                    new Size(ExtentValue, Owner.Panel.DesiredSize.Height);
+                if (IsLogicalScrollEnabled)
```

Comment:
The checks here for `IsLogicalScrollEnabled` aren't strictly needed, but they stop exceptions being thrown when viewing a non-virtualized `ItemsPresenter` in DevTools.